src/Event/ReadModel/JSONLD/OrganizerServiceInterface.php

```diff
@@ -13,12 +15,7 @@
 interface OrganizerServiceInterface
 {
     /**
-     * Gets the JSON-LD structure of an Organizer.
-     *
-     * @param string $organizerId
-     *   Id of the Organizer.
-     *
-     * @return \stdClass
+     * @return stdClass
```

Comment:
I'd love to, but the implementation somehow returns an array at one point and I was very hesitant to change that.

Reply:
Yeah, let's leave it like this for now. It needs other refactoring as well anyway because right now this is super confusing with `src/OrganizerService` which doesn't even implement this interface but `EntityServiceInterface`.